tests/backend/test_postselection.py

```diff
@@ -67,6 +67,6 @@ def test_measure_fock(setup_backend, cutoff, batch_size):
         photons_out = sum(res1 + res2)
 
         if batch_size is not None:
-            total_photons = np.tile(total_photons, batch_size)
+            total_photons = np.tile(total_photons, batch_size).sum()
```

Comment:
was this a bug before?

Reply:
Ah, thanks for pointing it out. It's (sort of) a misunderstanding of the test from my side, although I believe this should work as intended as well. 

The issues is a few lines up, with the line `photons_out = sum(res1 + res2)`. Earlier, the returns would have been either lists or tuples (for the `tf` backend) and thus the `+` would act as an append. Now the returns are arrays or tensors, meaning that the `+` would simply add the values at each index. Thus, either removing the `sum(...)` from line 67 or adding a `.sum()` as done here would solve the issue. :slightly_smiling_face: